kolibri/core/auth/api.py

```
@@ -6,6 +6,7 @@
 from uuid import UUID
 from uuid import uuid4
 
+import requests
```

Ruff (0.11.9)

`requests` imported but unused

Remove unused import: `requests`

(F401)